src/OSPSuite.Core/Import/Column.cs

```diff
@@ -60,5 +61,15 @@ public override string ToString()
       {
          return $"{Name} [{Unit}]";
       }
+
+      //unit is missing if the SelectedUnit is undefined and 1. the error is not geometric or 2. the mapped column is empty
+      public bool MissingUnitMapping()
+      {
```

Comment:
This has moved, probably where it belongs, but this is still super hard to read. 
Or do you find this expression acceptable?
Asking @Yuri05 for feedback here. In my opinion, this was accepted by mistake previously

Reply:
@msevestre I have updated the comments and removed an unnecessary check in the second case. Unofrtunately in the first case we need both checks: we just need to maske sure the Error is not standard geometric deviation, but we also need to check for null, otherwise we get an exception in startup. I would say in this updated version it should not be hard to read....

Reply:
I would extract `(ErrorStdDev == null || ErrorStdDev.Equals(Constants.STD_DEV_ARITHMETIC))` into a new method `IsArithmeticStd `or `IsArithmeticError`
(and may be also put some comments there e.g. explaining why `ErrorStdDev == null` means "arithmetic".)

Also I would rename the method to `IsUnitMappingMissing()`

Reply:
I think the problem is also the lack of consistency within the method. Sometimes we use ==, sometimes we use Equals, the comment specifies `geometric` but we are checking for arithmetic. 

We need to do ourselves favors and stop those multiple combinations of `&&`  `||`  `((()` etc.. There is absolutely no reason to have this complexity and also no reason to try to return all in one expression. The code should read as a story, and for me it does not read at all